### ISSUE TITLE
Run Command Shortcut (Ctrl + .) does't work anymore on Linux

### ISSUE THREAD

**Issue by Gugi264** [silverbulletmd/silverbullet#616]
Since the last update (0.5.11) the (Ctrl + .) command shortcut to open the "Run Command" page doesnt work anymore. The other Shortcut (Ctrl + /) still works fine.

When I use (Ctrl + .) on Linux it prints the letter "e" while using Firefox. No problem when using Vivaldi (Chrome based browser).
On Windows it works on Firefox as expected.

**Comment by zefhemel** [comment]
Could you try this? Switch to the [edge build](https://community.silverbullet.md/t/living-on-the-edge-builds/27), and put this in your SETTINGS page:

```
shortcuts:
- command: "{[Open Command Palette]}"
  key: "Ctrl-."
```

On my Linux Firefox (118.0.1) Ctrl-. works fine. But perhaps it's a keyboard type issue? Not sure what changed here.

**Comment by Gugi264** [comment]
Sadly that didnt change anything.

I tried incognito mode (didnt change anything).

I also tried to another key (Ctrl-e and Ctrl-y) for the command palette, neither of which were working Firefox or vivaldi.
Meanwhile Ctrl-. works consistently on vivaldi (even when it was changed to another key, not sure if this is intendet)

I am running Firefox 121.0.1

**Comment by zefhemel** [comment]
Odd. However with the settings change I described you can assign it to any other key you like.

**Comment by Gugi264** [comment]
Ok I tested it on a similar linux pc with firefox, there it worked like intended. Seems like a weird problem with my setup.

**Comment by Gugi264** [comment]
I found a solution, so if anyone else runs into the same problem, take a look here:

[https://superuser.com/questions/1687342/what-is-my-x11-doing-with-ctrl-period-and-how-can-i-disable-it](https://superuser.com/questions/1687342/what-is-my-x11-doing-with-ctrl-period-and-how-can-i-disable-it)

tldr: keyboard shortcut from gnome overwrites seems to catch ctrl+. before firefox (apparently doesn't happen for chromium). To disable run:  

> gsettings set org.freedesktop.ibus.panel.emoji hotkey "@as []"

fixed the issue
